nautobot/extras/tests/test_models.py

```diff
@@ -711,6 +711,12 @@ def test_defaults(self):
 
     def test_clean_overrides(self):
         """Verify that cleaning resets non-overridden fields to their appropriate default values."""
+
+        # Can't use validated save cause this field would be ignored
+        self.local_job.has_sensitive_variables = True
+        self.local_job.save()
+        self.local_job.refresh_from_db()
```

Comment:
Not sure I understand why this is needed for this specific field but not for any others? Is it an issue with the specific `local_job` that we're using or something?

Reply:
`local_job`("TestPass") `has_sensitive_variables` is `False` and `approval_required` is `False`. 
Running the test to override both fields to `True` would cause an error. That's why `has_sensitive_variables` should be set to True, then override it to False while `approval_required` to True.

Reply:
Can we use a different Job for this test then? One that inherently has `has_sensitive_variables = True`? Would be a more realistic test.